html/Axit/js/main.js

```diff
@@ -1,3 +1,6 @@
-import {addTabSwitchingEffect} from './tabs/tab-switching.js';
+/* eslint-disable max-len */
+import {addContactFormValidation} from './validation/contact-validation/contact-form-validation.js';
+import {addBannerFormValidation} from './validation/banner-validation/banner-form-validation.js';
```

Comment:
it is more registration form, as banner it is place, and place of this form can change